phoenix-core/src/main/java/org/apache/phoenix/compile/UpsertCompiler.java

```diff
@@ -187,6 +213,9 @@ public static MutationState upsertSelect(StatementContext childContext, TableRef
         int maxSizeBytes =
                 services.getProps().getInt(QueryServices.MAX_MUTATION_SIZE_BYTES_ATTRIB,
                     QueryServicesOptions.DEFAULT_MAX_MUTATION_SIZE_BYTES);
+        int maxCellSizeBytes =
```

Comment:
I don't understand why we need another config for this. Introducing a Phoenix level config to mean the same thing as an existing hbase-level config will lead to confusion and now we have to make sure that this config is updated in case the main hbase config is updated. Why not just use 
`hbase.client.keyvalue.maxsize` or `hbase.server.keyvalue.maxsize` ?